### PR TITLE
Add case for non cordova environments

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Fix problem with serve command. We are missing init call.

### CODE REVIEW

**Comment by StephenCoady** [comment]
:eyes:

**Comment by wtrocki** [comment]
Verified on IOS.

**Comment by wtrocki** [comment]
I have verified that for every platform. Suddenly it stopped working. 
Going to investigate server side configuration to see how things could change.

FYI @TommyJ1994

File: src/app/main.ts [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+  initAuth();
+}
+
 /**
  * Initializes Auth auth and creates main angular context
  * This will reload angular context again
  */
- function initAuth() {
+function initAuth() {
   // Ensure that Auth is init before Angular to prevent Redirect looping issues
```
Comment (TommyJ1994) [inline src/app/main.ts]: I think we can remove this Angular note now?

**Comment by TommyJ1994** [comment]
@wtrocki The web origins don't need to be wildcarded in the Keycloak. So for the origin `localhost:8080` the entry in keycloak should be `localhost:8080` as ``localhost:8080/*` won't match the string compare of the incoming origin. I updated the web origins in Keycloak for the client-app. Emulator/Browser work as expected. Let me know if you still see the issue.

**Comment by ciaranRoche** [comment]
Re-verified this, works in both browser and emulator, nice one @TommyJ1994

**Pull request merged**